common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -246,7 +244,7 @@ private DefaultResourceLeak track0(T obj) {
         }
 
         if (level.ordinal() < Level.PARANOID.ordinal()) {
-            if ((ThreadLocalRandom.current().nextInt(0, samplingInterval)) == 0) {
```

Comment:
its too bad we lose this method, but I guess its not worth supporting a new wrapper API just to expose this.